From 1abc4f9b179187d83ad153edd48edc0e27acc35d Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Wed, 26 Jun 2019 12:20:07 +0300
Subject: [PATCH] [SW-0] pll: remove skip pll

It is not needed anymore now that we passed bringup

This strap will be used to set i2c slave mode of operation

Change-Id: I468b296be492801b839629691ebbe845fdb6e72c
---

diff --git a/gaudi/asic.c b/gaudi/asic.c
index 2a65bcc..7decdd4 100644
--- a/gaudi/asic.c
+++ b/gaudi/asic.c
@@ -289,10 +289,6 @@
 int hl_pll_get_cfg(struct hl_pll_cfg **hl_pll_cfg)
 {
 	int pll_num = ARRAY_SIZE(gaudi_pll_cfg);
-	int boot_strap = regmap_readl(mmPSOC_GLOBAL_CONF_BOOT_STRAP_PINS);
-
-	if (boot_strap & SKIP_PLL_INIT)
-		return 0;
 
 	*hl_pll_cfg = gaudi_pll_cfg;
 
diff --git a/gaudi/common.h b/gaudi/common.h
index 0874123..e43837b 100644
--- a/gaudi/common.h
+++ b/gaudi/common.h
@@ -5,9 +5,6 @@
  * SPDX-License-Identifier:     GPL-2.0+
  */
 
-/* Use DumpSel[9] to skip PLL initialization */
-#define SKIP_PLL_INIT BIT(17)
-
 #define mmDMA_W_PLL_BASE		0x487000
 #define mmIF_W_PLL_BASE			0x488000
 #define mmIF_E_PLL_BASE			0x4A7000
